tests/Microsoft.Identity.Test.Unit/PublicApiTests/PublicClientApplicationTestsWithB2C.cs

```diff
@@ -131,6 +131,23 @@ public async Task B2CAcquireTokenWithValidateAuthorityTrueAndRandomAuthorityTest
             }
         }
 
+        [TestMethod]
+        [TestCategory("B2C")]
+        public void B2CIgnoreWithTenantId_Success()
+        {
+            using (var harness = base.CreateTestHarness())
+            {
+                PublicClientApplication app = PublicClientApplicationBuilder.Create(TestConstants.ClientId)
+                    .WithAuthority(new Uri(TestConstants.B2CCustomDomain), true)
```

Comment:
B2C have custom domain authorities like this one but also `TestConstants.B2CAuthority`